### PR TITLE
Surround top-level function and class definitions with two blank lines.

### PR DESCRIPTION
**Pull request by joren485**
##### SUMMARY

PEP8 says:
> Surround top-level function and class definitions with two blank lines.

This PR implements that.

##### ISSUE TYPE

 - Bugfix Pull Request

##### ADDITIONAL INFORMATION
I do not know if `Bugfix Pull Request` is the right `ISSUE TYPE`, but it comes closest of the four options.

### CODE REVIEW

**Comment by joren485** [comment]
ready_for_review

File: lib/ansible/modules/network/netvisor/pn_vrouter.py [hunk 1]
```diff
@@ -411,6 +411,7 @@ def main():
     run_cli(module, cli)
 
 # AnsibleModule boilerplate
+
 from ansible.module_utils.basic import AnsibleModule
```
Comment (abadger) [inline lib/ansible/modules/network/netvisor/pn_vrouter.py]: I'm okay with you moving these sorts of imports up with the other imports near the top of the file (as you do in some of the other files)

File: lib/ansible/module_utils/database.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ class UnclosedQuoteError(SQLParseError):
 # maps a type of identifier to the maximum number of dot levels that are
 # allowed to specify that identifier.  For example, a database column can be
 # specified by up to 4 levels: database.schema.table.column
+
```
Comment (abadger) [inline lib/ansible/module_utils/database.py]: In general comments go with the code below them.  When a double blank line is called for, I would therefore put both blank lines above the comment.

File: lib/ansible/utils/hashing.py [hunk 3]
```diff
@@ -70,6 +70,7 @@ def secure_hash(filename, hash_func=sha1):
     return digest.hexdigest()
 
 # The checksum algorithm must match with the algorithm in ShellModule.checksum() method
+
```
Comment (joren485) [inline lib/ansible/utils/hashing.py]: This newline needs to be moved above the comment

File: lib/ansible/plugins/callback/dense.py [hunk 4]
```diff
@@ -494,6 +494,7 @@ def v2_playbook_on_stats(self, stats):
             )
 
 # When using -vv or higher, simply do the default action
+
```
Comment (joren485) [inline lib/ansible/plugins/callback/dense.py]: This newline needs to be moved above the comment

**Comment by abadger** [comment]
In addition to the two notes (which apply in several places) that I left above, you should also enable this test in the pep8 test: https://github.com/ansible/ansible/blob/devel/test/sanity/pep8/current-ignore.txt  E305.

I also have to get permission to merge bulk style changes from the other committers.  In this case, I think that it should be allowed as it doesn't change attribution and the changes are such that I can easily review them all (I just did).  However, style-only changes being sensitive, I need to seek out that permission first.

File: lib/ansible/modules/web_infrastructure/jira.py [hunk 5]
```diff
@@ -368,6 +368,7 @@ def link(restbase, user, passwd, params):
     return ret
 
 # Some parameters are required depending on the operation:
+
```
Comment (joren485) [inline lib/ansible/modules/web_infrastructure/jira.py]: This newline needs to be moved above the comment

File: lib/ansible/modules/web_infrastructure/apache2_module.py [hunk 6]
```diff
@@ -260,6 +260,7 @@ def main():
         _set_state(module, module.params['state'])
 
 # import module snippets
+
```
Comment (joren485) [inline lib/ansible/modules/web_infrastructure/apache2_module.py]: This newline needs to be moved above the comment

File: lib/ansible/modules/system/awall.py [hunk 7]
```diff
@@ -149,5 +149,6 @@ def main():
     module.fail_json(msg="no action defined")
 
 # import module snippets
+
```
Comment (joren485) [inline lib/ansible/modules/system/awall.py]: This newline needs to be moved above the comment

File: lib/ansible/modules/packaging/os/zypper.py [hunk 8]
```diff
@@ -523,6 +523,7 @@ def main():
     module.exit_json(name=name, state=state, update_cache=update_cache, **retvals)
 
 # import module snippets
+
```
Comment (joren485) [inline lib/ansible/modules/packaging/os/zypper.py]: This newline needs to be moved above the comment

File: lib/ansible/modules/network/netvisor/pn_vrouterlbif.py [hunk 9]
```diff
@@ -322,6 +322,7 @@ def main():
     run_cli(module, cli)
 
 # Ansible boiler-plate
+
```
Comment (joren485) [inline lib/ansible/modules/network/netvisor/pn_vrouterlbif.py]: This newline needs to be moved above the comment

**Comment by joren485** [comment]
@abadger The changes you suggested are made (in all relevant places).

**Comment by abadger** [comment]
So far, sivel and I are +1 on merging a fixed version of this. I'll try to gather more feedback/votes at the Tuesday IRC meeting since I know style changes have been a source of controversy in the past.

**Comment by samdoran** [comment]
👍

**Comment by abadger** [comment]
bcoca also gave his +1 at the meeting today.  No one opposed to it.  We're calling it approved.

The unstable CI test failure is unrelated to the PR so going to merge this to devel.

**Comment by abadger** [comment]
Merged to devel for the 2.7.0 release.

**Pull request merged**